W3C.Domain/UpdateService/UpdateServiceClient.cs

```diff
@@ -4,9 +4,11 @@
 using System.Net.Http;
 using System.Threading.Tasks;
 using W3C.Domain.UpdateService.Contracts;
+using W3C.Domain.Tracing;
 
 namespace W3C.Domain.UpdateService;
 
+[Trace]
```

Comment:
Is adding `HttpRequestMessage` to traces potentially problematic? Not sure what the `ToString()` would result in and whether it may include headers or something

Reply:
I will double check!

Reply:
<img width="1637" alt="image" src="https://github.com/user-attachments/assets/76c2e583-8651-481c-a79e-596c9671413d" />

So it seems like sensitive parameters are automatically redacted and the headers are not traced.